config/local/gateway-service.yml

```diff
@@ -24,6 +24,8 @@ eureka:
         initialInstanceInfoReplicationIntervalSeconds: 1
         registryFetchIntervalSeconds: 1
 spring:
+    config:
+        useLegacyProcessing: true
```

Comment:
is it necessary to set this parameter again in the local config?

Reply:
Without it there is a warning due to the `spring.profiles.include` line at the top of local config files.